vue3-apollo-quasar/tests/unit/components/PaginationButtons.spec.ts

```diff
@@ -1,33 +1,80 @@
 import { mount } from '@vue/test-utils';
 import { PaginationButtons } from '@/components';
 
-describe('PaginationButtons', () => {
-  const wrapper = mount(PaginationButtons, {
+let wrapper;
+
+beforeAll(() => {
+  wrapper = mount(PaginationButtons, {
     props: {
       isPrevActive: false,
       isNextActive: true,
     },
   });
-
-  const pagination_btns = wrapper.findAll('.pagination_btn');
+});
+describe('PaginationButtons mounting', () => {
```

Comment:
Tests look good. Please check out the previous comment though

Reply:
@hdJerry , I understand that this is the pagination PR but if we keep pushing to do it (switch to quasar component) later (as we did previously) it'll never happen